storage/client/file/src/main/java/com/azure/storage/file/implementation/DirectorysImpl.java

```diff
@@ -55,23 +56,23 @@ public DirectorysImpl(AzureFileStorageImpl client) {
     }
 
     /**
-     * The interface defining all the services for Directorys to be used by the
-     * proxy service to perform REST calls.
+     * The interface defining all the services for AzureFileStorageDirectorys
+     * to be used by the proxy service to perform REST calls.
      */
     @Host("{url}")
-    @Service("Storage File Directories")
+    @ServiceInterface(name = "AzureFileStorageDirectorys")
     private interface DirectorysService {
-        @PUT("{shareName}/{directory}")
+        @Put("{shareName}/{directory}")
```

Comment:
Can we rename to directoryPath in consist with the API level?
The same question applies to FilesImpl